config/configmap/inferenceservice.yaml

```diff
@@ -87,6 +87,14 @@ data:
               "pmml"
             ],
             "multiModelServer": "false"
+        },
+        "lightgbm": {
+            "image": "kfserving/lgbserver",
+            "defaultImageVersion": "v0.5.0-rc0",
+            "supportedFrameworks": [
+              "lightgbm"
+            ],
+            "multiModelServer": "false"
```

Comment:
if we follow xgboost's code you should be able to support Multi model serving

Reply:
ok